zebra-test/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "zebra-test"
-version = "1.0.0-beta.33"
+version = "1.0.0-beta.34"
```

Optional:

We could skip bumping the version for this crate since there haven't been any code or dependency changes.

It's also reasonable to bump the version to keep it consistent with the other crates.

This PR looks good either way.

```suggestion
version = "1.0.0-beta.33"
```

---

Yea, i decided to update everything because:
- The command in the release drafter is one to update all the betas.
- I think we want to keep all the betas the same number.
- With that , we could still skip the upgrade of `tower-*` crates but i think it might be confusing to upgrade some crates and not others.

Also for future hotfixes i think we should always bump everything or change the drafter.